definitions/procedures/backup/online/safety_confirmation.rb

```diff
@@ -0,0 +1,20 @@
+module Procedures::Backup::Online
```

Comment:
Shouldn't this be a check rather than a procedure? I have to a similar thing in restore so wondering where to put this type of check

Reply:
I was not sure either. As I understand it checks are testing status of the system. If it fails there may be attempt to fix that via some remediation step or the scenario fail. This was not exact match so I used procedures. But I agree these feel more like checks and I can look into how we can modify checks to ask for confirmation from the user.

Reply:
Thanks, I'll put it in procedures for now as well, if you wind up modifying checks for user input, give me a ping and I'll update as well!